<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testPaper</class>
 <widget class="QDialog" name="testPaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelCourseName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>C++</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Q1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblQ1">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>40</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb1A">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb1B">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb1C">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb1D">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>140</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb2C">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb2A">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="lblQ2">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>200</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb2B">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Q2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb2D">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb3C">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb3A">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="lblQ3">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>360</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb3B">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>430</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Q3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb3D">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>470</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb4C">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>630</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_4</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="rb4A">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_4</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="lblQ4">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>520</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb4B">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>590</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_4</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Q4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb4D">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>630</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_4</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>680</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>690</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
